### PR TITLE
fixing some quick typos

### PR DESCRIPTION
**Pull request by lamek**

### CODE REVIEW

File: modules/nw-multus-advanced-annotations.adoc [hunk 1]
```diff
@@ -150,7 +150,7 @@ and IP address using the macvlan CNI plug-in:
 
 <2> The `capabilities` key denotes that a request is being made to enable the static IP functionality of a CNI plugins runtime configuration capabilities
 
-<3> The `master` field is specific to the macvlan plugin (TODO: link to see macvlan docs)
+<3> The `master` field is specific to the macvlan plugin
```
Comment (jboxman) [inline modules/nw-multus-advanced-annotations.adoc]: s/plugin/plug-in./

**Comment by jboxman** [comment]
Thanks. I haven't had time to thoroughly go through these, as I'm focusing on the API docs right now.

File: modules/nw-multus-advanced-annotations.adoc (hunk 1 above)
Comment (lamek) [reply modules/nw-multus-advanced-annotations.adoc]: Got it.

**Pull request merged**